fab/fabfile.py

```diff
@@ -462,8 +453,8 @@ def update_code(use_current_release=False):
 
     with cd(env.code_root if not use_current_release else env.code_current):
         sudo('git remote prune origin')
-        sudo('git fetch')
-        sudo("git submodule foreach 'git fetch'")
+        sudo('git fetch origin {}'.format(env.code_branch))
+        sudo("git submodule foreach 'git fetch origin {}'".format(env.code_branch))
```

Comment:
would this be a problem if you ever wanted to deploy a commit that wasn't on master to a submodule? or if you hotfix deploy off a branch that doesn't exist.